packages/@ember/-internals/runtime/tests/system/object/create_test.js

```diff
@@ -197,7 +211,9 @@ moduleFor(
       assert.equal(result.foo.bar, 'foo');
     }
 
-    ['@test does raise deprecation if descriptor is a getter and not equal to the implicit deprecation'](assert) {
+    ['@test does not raise deprecation if descriptor is a getter and not equal to the implicit deprecation'](
```

Comment:
What this change intended?  Based on the code changes, it seems so?

Reply:
Yes, after thinking about it some more my conclusion was that we can't really assert about getters, specifically because when a user defines a getter, they also must define a setter. Otherwise, the implicit injection will do nothing (or error, if JS strict mode is enabled). With that in mind, its possible that the user could define a setter that swallows the implicit injection, handling the deprecated behavior essentially.

Happy to discuss further if that doesn't make sense